modules/custom-status/lib/components/modals/create-edit-custom-status-modal.js

```diff
@@ -55,8 +53,8 @@ export default function CreateEditCustomStatusModal( {
 	// Modal properties
 	const [ error, setError ] = useState( null );
 	const [ isRequesting, setIsRequesting ] = useState( false );
-	const [ isRestrictedSectionVisible, setIsRestrictedSectionVisible ] = useState(
-		requiredUsers.length > 0 || requiredMetadatas.length > 0
+	const [ areRestrictedUsersSet, setAreRestrictedUsersSet ] = useState(
```

Comment:
Not the biggest fan of setting text here to control the radio control, so happy to change this. This seemed like the best way imo for now.